### PR TITLE
Add incidents to cloud index page

### PR DESCRIPTION
**Pull request by discdiver**
### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [X] This pull request includes tests or only affects documentation.
- [X] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [ ] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed

### CODE REVIEW

File: docs/cloud/index.md [hunk 1]
```diff
@@ -97,6 +97,11 @@ Prefect Cloud allows you to see your [events](/cloud/events/). Events provide in
 Prefect Cloud [automations](/cloud/automations/) provide additional notification capabilities beyond those in a self-hosted open-source Prefect server.
 Automations also enable you to create event-driven workflows.
 
+## Incidents <span class="badge pro"></span> <span class="badge enterprise"></span> <span class="badge beta"/>
+
+Prefect Cloud's [incidents](/concepts/incidents/)  help teams identify, rectify, and document issues in mission-critical workflows. 
+Incidents are formal declarations of disruptions to workflows, and provide a temporary control plane for collaboratively managing those disruptions.
```
Comment (billpalombi) [inline docs/cloud/index.md:103]:
```suggestion
Incidents are formal declarations of disruptions to a workspace. With Automations, activity in that workspace can be paused when an incident is created and resumed when it is resolved.
```

Comment (billpalombi) [reply docs/cloud/index.md:103]: I know we've used "temporary control plane" internally, but I find that very abstract. I think its clearer to spell it out for users.

**Comment by discdiver** [comment]
Adding badges, updating language, moving to cloud section, and adding redirect.

**Comment by discdiver** [comment]
Screenshot 2024-01-03 at 10 49 17 AM

**Review by serinamarie — APPROVED** [review]
lgtm

**Review by billpalombi — APPROVED** [review]
lgtm

**Pull request merged**